src/main/java/com/suse/saltstack/netapi/utils/LogUtils.java

```diff
@@ -0,0 +1,297 @@
+package com.suse.saltstack.netapi.utils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utility class for handling logger instances and printing parsed messages according to caller method<br/>
+ * Logger instances will be created using reflection but the performance effect will be minimum since<br/>
+ * instances handled by local cache. <br/>
+ * <br/>
+ * <b>Usage:</b><br/>
+ *       In your class simply call <b><code>LoggingUtils.debug("hello logger");</code></b><br/>
+ * Simply edit log4j.properties file from project folder to handle logging level.<br/>
+ * @author Burak Sarac, burak at linux dot com, 20/02/2015 00:47
```

Comment:
@renner: do we want authorship and license notices in comments? Personally I think they are not very useful, since `git blame` already does that in a more precise way.